moveit_ros/planning/kinematics_plugin_loader/include/moveit/kinematics_plugin_loader/kinematics_plugin_loader.h

```diff
@@ -54,27 +55,8 @@ class KinematicsPluginLoader
       found. This is passed to the kinematics solver initialization as
       well as used to read the SRDF document when needed. */
   KinematicsPluginLoader(const rclcpp::Node::SharedPtr& node,
-                         const std::string& robot_description = "robot_description",
-                         double default_search_resolution = 0.0)
-    : node_(node), robot_description_(robot_description), default_search_resolution_(default_search_resolution)
-  {
-  }
-
-  /** \brief Use a default kinematics solver (\e solver_plugin) for
-      all the groups in the robot model. The default timeout for the
-      solver is \e solve_timeout and the default number of IK attempts
-      is \e ik_attempts. Take node as an argument and as optional argument the name of the ROS
-      parameter under which the robot description can be found. This
-      is passed to the kinematics solver initialization as well as
-      used to read the SRDF document when needed. */
-  KinematicsPluginLoader(const rclcpp::Node::SharedPtr& node, const std::string& solver_plugin, double solve_timeout,
-                         const std::string& robot_description = "robot_description",
-                         double default_search_resolution = 0.0)
```

Comment:
Deleting this constructor since it's not used anywhere. All default kinematics parameters are listed in `kinematics_parameters.yaml`